Apps/WebClient/src/ClientApp/app/components/navmenu/navHeader.vue

```diff
@@ -2,7 +2,7 @@
   <b-navbar toggleable="lg" type="dark">
     <!-- Brand -->
     <b-navbar-brand>
-      <router-link to="/">
```

Comment:
Without this there is no way to navigate to the landing page once logged in. Is that intentional?

Reply:
Yes this was intentional